cogs/commands/quotes.py

```diff
@@ -1,73 +1,92 @@
 import logging
 import re
 from datetime import datetime
-from typing import List
+from enum import Enum
+from typing import Optional
 
 from discord import AllowedMentions
 from discord.ext import commands
 from discord.ext.commands import Bot, Context, Converter
-from discord.ext.commands.converter import clean_content
 from sqlalchemy.exc import SQLAlchemyError
 from sqlalchemy.sql import func
-from sqlalchemy.sql.operators import is_
 from sqlalchemy_utils import ScalarListException
 
-from models import Quote, QuoteOptouts, db_session, quote
+from models import db_session
+from models.quote import *
 from utils import (
     get_database_user,
     get_name_string,
     is_compsoc_exec_in_guild,
     user_is_irc_bot,
 )
-from utils.mentions import Mention, MentionConverter, MentionType, parse_mention
+from utils.mentions import *
 
 LONG_HELP_TEXT = """
 Pull a random quote. Pull quotes by ID using "#ID", by author using "@username", or by topic by entering plain text
 """
 SHORT_HELP_TEXT = """Record and manage quotes attributed to authors"""
 
 
+class QuoteError(Enum):
+    BAD_FORMAT = 1
+    NOT_PERMITTED = (2,)
+    NOT_FOUND = (3,)
+    OPTED_OUT = (4,)
+    DB_ERROR = (5,)
```

Comment:
Random tuples here. You may want to use:
```py
from enum import auto, Enum, unique

@unique
class QuoteError(Enum):
    BAD_FORMAT = auto()
    NOT_PERMITTED = auto()
    # and so forth
```
I'm guessing you added trailing commas as would be the case in enums from another language, such as Rust or C#.